<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logWindow</class>
 <widget class="QWidget" name="logWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(0, 170, 255)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="head">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>头像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chat_choose_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#chat_choose_button:!hover
{
border-style:inset;
border-radius: 5px;
background-color:rgb(0, 170, 255);
}
#chat_choose_button:hover
{
border-radius: 5px;
background-color: rgb(1, 142, 235);
}
#chat_choose_button:pressed{
border-radius: 5px;
background-color: rgb(0, 130, 230);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../image.qrc">
      <normaloff>:/new/icon/image/chat_icon.png</normaloff>:/new/icon/image/chat_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pyq_choose_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pyq_choose_button:!hover
{
border-style:inset;
border-radius: 5px;
background-color:rgb(0, 170, 255);
}
#pyq_choose_button:hover
{
border-radius: 5px;
background-color: rgb(1, 142, 235);
}
#pyq_choose_button:pressed{
border-radius: 5px;
background-color: rgb(0, 130, 230);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../image.qrc">
      <normaloff>:/new/icon/image/pyq_icon.png</normaloff>:/new/icon/image/pyq_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="item_choose_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#item_choose_button:!hover
{
border-style:inset;
border-radius: 5px;
background-color:rgb(0, 170, 255);
}
#item_choose_button:hover
{
border-radius: 5px;
background-color: rgb(1, 142, 235);
}
#item_choose_button:pressed{
border-radius: 5px;
background-color: rgb(0, 130, 230);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../image.qrc">
      <normaloff>:/new/icon/image/item_icon.png</normaloff>:/new/icon/image/item_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>37</width>
      <height>37</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="set_choose_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#set_choose_button:!hover
{
border-style:inset;
border-radius: 5px;
background-color:rgb(0, 170, 255);
}
#set_choose_button:hover
{
border-radius: 5px;
background-color: rgb(1, 142, 235);
}
#set_choose_button:pressed{
border-radius: 5px;
background-color: rgb(0, 130, 230);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../image.qrc">
      <normaloff>:/new/icon/image/set_icon.png</normaloff>:/new/icon/image/set_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>46</width>
      <height>46</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>981</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pyq_page
{
background-image: url(:/new/bg/image/background-pyq.png);
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="chat_page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>15</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>聊天界面</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="friendsArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>251</width>
       <height>641</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>249</width>
        <height>639</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPlainTextEdit" name="texteditInput">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>500</y>
       <width>691</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>11</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:transparent</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="send_button">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>660</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#send_button:!hover
{
border-radius: 5px;
background-color:rgb(0, 170, 255);
}
#send_button:hover
{
border-radius: 5px;
background-color: rgb(85, 170, 255);
}
QPushButton:pressed{
border-radius: 5px;
background-color: rgb(0, 130, 230);
}</string>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pic_send">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>460</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:!hover
{
border-radius: 5px;
background-color:rgb(0, 170, 255);
}
QPushButton:hover
{
border-radius: 5px;
background-color: rgb(85, 170, 255);
}
QPushButton:pressed{
border-radius: 5px;
background-color: rgb(0, 130, 230);
}</string>
     </property>
     <property name="text">
      <string>图片</string>
     </property>
    </widget>
    <widget class="QPushButton" name="video_send">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>460</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:!hover
{
border-radius: 5px;
background-color:rgb(0, 170, 255);
}
QPushButton:hover
{
border-radius: 5px;
background-color: rgb(85, 170, 255);
}
QPushButton:pressed{
border-radius: 5px;
background-color: rgb(0, 130, 230);
}</string>
     </property>
     <property name="text">
      <string>视频</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="chat_stack_widget">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>731</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>110</y>
         <width>391</width>
         <height>191</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>AR PShinChanPOP</family>
         <pointsize>48</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt; color:#b5b5b5;&quot;&gt;We Work&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>0</y>
       <width>16</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="chat_title">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>16</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>21</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:!hover
{
border-radius: 5px;
background-color:rgb(0, 170, 255);
}
QPushButton:hover
{
border-radius: 5px;
background-color: rgb(85, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="video_send_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>460</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:!hover
{
border-radius: 5px;
background-color:rgb(0, 170, 255);
}
QPushButton:hover
{
border-radius: 5px;
background-color: rgb(85, 170, 255);
}
QPushButton:pressed{
border-radius: 5px;
background-color: rgb(0, 130, 230);
}</string>
     </property>
     <property name="text">
      <string>文件</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pyq_page">
    <widget class="QScrollArea" name="scrollArea_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>487</width>
       <height>641</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#scrollAreaWidgetContents_3{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0.614, stop:0 rgba(156, 190, 207, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>485</width>
        <height>639</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>朋友圈</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="item_page">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>210</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>待办界面</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>设置界面</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
